### PR TITLE
[#159554875] Configure CircleCI.

### PR DESCRIPTION
**Pull request by duozersk**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .PHONY: default pull up stop down exec-www-data exec exec-root drush \
 prepare prepare-composer prepare-files \
-install
+install test2
```
Comment (kalabro) [inline Makefile]: What is test2?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            sed -i '/-dev-macos-/s/^/# /g' \.env
+
+      - run:
+          name: Make local environment
+          command: make install
+          no_output_timeout: 20m
+
+      - run:
+          name: Add hosts into /etc/hosts
+          command: echo 127.0.0.1 falcon.docker.localhost | sudo tee -a /etc/hosts
```
Comment (kalabro) [inline .circleci/config.yml:33]: I didn't know tee command 👍

File: docs/index.rst [hunk 3]
```diff
@@ -7,12 +7,12 @@ Falcon is an open source project for charities.
 
 Quick start
 -----------
-Follow instractions in :doc:`install` section.
+Follow instructions in :doc:`install` section.
```
Comment (kalabro) [inline docs/index.rst]: Thanks for fixing those typos 😊

File: Makefile (hunk 1 above)
Comment (duozersk) [reply Makefile]: To be removed :)

File: .circleci/config.yml (hunk 2 above)
Comment (duozersk) [reply .circleci/config.yml:33]: Me too, me too :)

**Review by kalabro — APPROVED** [review]

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+version: 2
+
+jobs:
+  build_and_test:
+
+    machine:
```
Comment (spleshka) [inline .circleci/config.yml:6]: @duozersk can you explain what is being done here and why we use this image?

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      - run:
+          name: Add hosts into /etc/hosts
+          command: echo 127.0.0.1 falcon.docker.localhost | sudo tee -a /etc/hosts
+
+      - run:
+          name: Test frontpage
+          command: curl -sSf http://falcon.docker.localhost | grep '<title>Log in | falcon</title>'
+
+      - run:
+          name: Check drush status
```
Comment (spleshka) [inline .circleci/config.yml:40]: I think drush status will throw no error even if there is no db connection?

File: .circleci/config.yml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+          command: make drush status
+
+      - run:
+          name: Cleanup local environment
```
Comment (spleshka) [inline .circleci/config.yml:44]: What is the reason for this command here?

Comment (duozersk) [reply .circleci/config.yml:44]: Just to keep our hands clean; can be removed for sure (as the whole environment is going to be destroyed anyway).

File: .circleci/config.yml (hunk 5 above)
Comment (duozersk) [reply .circleci/config.yml:40]: Don't know; these 2 checks are just some trivial things to actually see that environment is working - just as described in the tasks for this story. Kate is working on implementing some real "example" tests now.

File: .circleci/config.yml (hunk 4 above)
Comment (duozersk) [reply .circleci/config.yml:6]: We need to use `machine` executor to run docker-compose based builds.
See https://circleci.com/docs/2.0/docker-compose/#using-docker-compose-with-machine-executor and https://circleci.com/docs/2.0/executor-types/#using-machine

**Pull request merged**